problems/objects.js

```diff
@@ -36,7 +40,10 @@ const appleCount = () => {};
  * @returns {Object} The updated object.
  */
 
-const eveAppleSet = () => {};
+const eveAppleSet = (appleCountByName, appleCount) => {
+  appleCountByName.Eve = appleCount
```

Comment:
Love that you're doing both bracket and dot notation because it shows us you understand both!